production-deployment-using-tiup.md

```diff
@@ -133,6 +133,52 @@ To prepare the TiUP offline component package, manually pack an offline componen
 
         `tidb-community-server-${version}-linux-amd64.tar.gz` is an independent offline environment package.
 
+3. Customize the offline mirror, or adjust the contents of the existing offline  mirror.
+
+    If the existing offline mirrors does not meet your specific needs, or if you want to make adjustments to the mirror (like adding a new version of a component), take the following process:
```

Comment:
I have revised the first line of this part because we don't provide the package that can be downloaded from our website, please take a look.

Reply:
```suggestion
    If you want to adjust the existing offline mirror (such as adding a new version of a component), take the following steps:
```